dotnet/DDB.Bindings/DroneDB.cs

```diff
@@ -47,19 +62,39 @@ public static string Init(string directory)
 
         public static List<Entry> Add(string ddbPath, string path, bool recursive = false)
         {
+
```

Comment:
Same, should this be moved to the lower level of the API?

Reply:
See https://github.com/DroneDB/DroneDB/pull/115#issuecomment-709561637